.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
```

Comment:
It would be nice if the default template was a bit more edited. We're mainly providing APIs so the "Go to", "Scroll down to" are not really in context.

I would change this section by something like this:

> For example, here are some steps that would be useful to us:
> 1. Initialize storage
> 2. Stop internet connection
> 3. Restore internet connection
> 4. The storage is not in sync

Reply:
I'll improve this one.

Reply:
I've changed the template using a single scenario for all the paragraphs.